src/gui/mainwindow.h

```diff
@@ -36,9 +36,14 @@
 #include <QVBoxLayout>
 #include <QWidget>
 #include <QtCore/QVariant>
+#include <opencv2/imgcodecs.hpp>
 
 #include "../intermediate_structures.h"
 #include "../posture_estimator.h"
+#include "opencv2/core.hpp"
+#include "opencv2/highgui/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/videoio.hpp"
```

Comment:
Are you sure we definitely need all of these imports?

Reply:
Good point i will check this now and update it accordingly!

Reply:
Actually since the makefile is pointing to opencv we don't need them at all.